frontend/app/services/EventbriteService.scala

```diff
@@ -143,7 +143,7 @@ object GuardianLiveEventService extends LiveService {
 
 object LocalEventService extends LiveService {
   val apiToken = Config.eventbriteLocalApiToken
-  val maxDiscountQuantityAvailable = 2 //TODO are these discounts correct for local?
+  val maxDiscountQuantityAvailable = 3
```

Comment:
You can use your local copy of the `/etc/gu/membership-keys.conf` file to make our Dev Eventbrite account supply the events for Guardian Live, so testing with Local events is unnecessary.

The free event tickets are only for Guardian Live Events, , so this value here should stay at 2